ee_tests/src/page_objects/login.page.ts

```diff
@@ -47,10 +47,14 @@ export class LoginPage extends BasePage {
     await this.passwordInput.enterText(password);
     await this.loginButton.clickWhenReady();
 
+    support.writeScreenshot('target/screenshots/login_1.png');
+
     this.debug('... Login: input details and click Login - OK');
 
     let mainDashboard = new MainDashboardPage()
     await mainDashboard.open()
```

Comment:
Although not related to this PR, please add semicolons to these two lines

Reply:
Done